docs/Usage.md

```diff
@@ -56,28 +58,32 @@ If you are using a separate Edge device, like a Raspberry Pi, you do not need to
 
     `sudo iotedgedev stop`
 
-    > You can avoid `sudo` if you are running IoT Edge Dev Tool outside Docker container, and:
-    > * You are on Windows or macOS.
-    > * You are on Linux, and you have followed the [post-installation steps for Linux](https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user) to allow running Docker commands without `sudo`.
+    You can avoid `sudo` if you are running IoT Edge Dev Tool outside Docker container, and:
+    * You are on Windows or macOS.
+    * You are on Linux, and you have followed the [post-installation steps for Linux](https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user) to allow running Docker commands without `sudo`.
 
 1. **Setup the IoT Edge Runtime**
-    
+
     1. Copy the device connection string from the `DEVICE_CONNECTION_STRING` environment variable in the `.env` file, [Azure Portal](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-register-device-portal#retrieve-the-connection-string), or [Azure CLI](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-register-device-cli#retrieve-the-connection-string).
     1. Follow [Edge Device Setup](edge-device-setup) to setup and start the IoT Edge Runtime with the copied device connection string
 
 1. **Push IoT Edge module images**
 
     `sudo iotedgedev push`
 
-    > You can avoid `sudo` if you are running IoT Edge Dev Tool outside Docker container, and:
-    > * You are on Windows or macOS.
-    > * You are on Linux, and you have followed the [post-installation steps for Linux](https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user) to allow running Docker commands without `sudo`.
+    You can avoid `sudo` if you are running IoT Edge Dev Tool outside Docker container, and:
+    * You are on Windows or macOS.
+    * You are on Linux, and you have followed the [post-installation steps for Linux](https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user) to allow running Docker commands without `sudo`.
 
 1. **Deploy modules to IoT Edge device**
 
     `iotedgedev deploy`
-    > You can also combine push and deploy with `iotedgedev push --deploy`
+    You can also combine push and deploy with `iotedgedev push --deploy`
+
+1. **Create deployments on IoT Hub**
```

Comment:
as iotedge uses layered deployment as a concept rather than iothub deployments, do you think we should add that term int he documentation explicitely?

Reply:
I've editing to the reference 'automatic deployments' (what they call it)